Game/templates/Game/result_all.html

```diff
@@ -49,36 +49,27 @@
         <hr class="my-5">
         <h3>Room : {{ room.room_id }}</h3>
         <div class="row justify-content-center">
-            {% comment %} <div class="col-md-6">
-                <div class="card">
-                        <div class="card-header">For Hamburger</div>
-                    <div class="card-body">
-                        <form action="" method="">
-                            <div class="col-md-6 offset-md-10">
-                                    <a href="{% url 'Game:home' %}" class="btn btn-secondary">
-                                        Back
-                                    </a>
-                            </div>
-                    </div>
-                    </form>
-                </div>
-            </div> {% endcomment %}
             <div class="col-md-3">
                 <div class="card">
                         <div class="card-header">Player info</div>
                     <div class="card-body">
-                        <form action="" method="">
+                            {% for each_player in all_player %}
```

Comment:
I will try to add the score of each player in the form class, so the player can easily see his/her result.